### ISSUE TITLE
Feature request: vim-like mouse-free control

### ISSUE THREAD

**Issue by CptnGreen** [stepan-mitkin/drakon_editor#36]
Доброго времени суток! Во-первых хочу сказать спасибо вам, Степан, за ваш редактор. Периодически использовал его для объяcнения ребятам программ на Си в кружке робототехники ("Лига Роботов", Москва). Сейчас сам решил подучиться программированию, активный пользователь vim; Как вы наверное знаете, идеология vim такова - не отрывать руки от клавиатуры, мышь - зло. И при этом хочется часто переключаться между vim и вашим редактором (чтобы писать сложный код, параллельно объясняя его себе). Возникает вопрос: есть ли возможность создать "vim-skin" для drakon editor, в котором всё делалось бы через клавиатуру? Какие возможно шаги на этом пути? Еще раз спасибо.

Hello! First of all, i want to thank developer for his work. I periodically use drakon editor to demonstrate C code on my robotics classes for kids. Now i'm actively learning myself with heavy use of vim editor which philosophy is "mouse is evil". I want to use both - drakon editor (concept) AND vim (actual writing) in parallel without using mouse of course. My question is: is there a way to write some vim-skin for drakon editor for us vim-users? What are the steps? Thank you again.

**Comment by einhander** [comment]
@CptnGreen   That about Emacs users? Maybe this feature request should be named as mouse free controll. What do you think? Vim or Emacs style it's mater of of taste. I'm +1 for vim scheme, through.
Может быть стоит всё же стоит реализовать саму возможность работы без мыши, а какой тип управления vim или Emacs, это дело десятое. Хотя я за vim подобную раскладку.

**Comment by CptnGreen** [comment]
@einhander, i agree, it would be ideal to have an opportunity to choose from at least three option here: (1) standard controls, (2) vim-like, (3) emacs-like

**Comment by krvkir** [comment]
+1 for customizable keyboard control layer capable of both emacs and vim keybindings. I am also heavy user of keyboard-only editors like Vim and Spacemacs, and I often use Drakon editor when I need to put down on paper some complex business logic or algorithm and talk about it with others (it helps a lot in this role!). It would be great to be able to work with Drakon editor mostly with a keyboard.
